packages/cli/test/integration.test.ts

```diff
@@ -21,10 +22,12 @@ import fs, {
 } from 'fs-extra';
 import logo from '../src/util/output/logo';
 import sleep from '../src/util/sleep';
-import pkg from '../package';
+import pkg from '../package.json';
 import prepareFixtures from './helpers/prepare';
 import { fetchTokenWithRetry } from '../../../test/lib/deployment/now-deploy';
 
+jest.setTimeout(60 * 1000);
```

Comment:
```suggestion
jest.setTimeout(3 * 60 * 1000);
```

Lets bump this up for now since there were several timeouts. In the future we can split up this file into multiple.

https://github.com/vercel/vercel/actions/runs/4369420942/jobs/7643198796